src/_nebari/stages/infrastructure/template/azure/modules/kubernetes/main.tf

```diff
@@ -4,6 +4,9 @@ resource "azurerm_kubernetes_cluster" "main" {
   location            = var.location
   resource_group_name = var.resource_group_name
   tags                = var.tags
+  api_server_access_profile {
+    authorized_ip_ranges = var.authorized_ip_ranges
+  }
```

Comment:
Does including this field here leads to a cluster recreation? e.g does it affects the running nodes at all?

Reply:
It shouldn't be I can test.

Reply:
All it does is update and attribute, no replacement needed. It also runs on the cluster, not on the node.